### PR TITLE
Fix tests and checkstyle

### PR DESCRIPTION
**Pull request by dzou**
Fix tests and checkstyle for Handler2 and add a wait for server startup in integration tests.

### CODE REVIEW

**Review by meltsufin — APPROVED** [review]
I have the same fix in my local using mock.

File: spring-cloud-function-adapters/spring-cloud-function-adapter-gcp/src/test/java/org/springframework/cloud/function/adapter/gcloud/integration/CloudFunctionServer.java [hunk 1]
```diff
@@ -76,6 +76,9 @@ protected void before() {
 
 		this.serverThread = new Thread(startServer);
 		this.serverThread.start();
+
+		// Wait for the server to start up.
+		Thread.sleep(1000);
```
Comment (dmitry-s) [inline spring-cloud-function-adapters/spring-cloud-function-adapter-gcp/src/test/java/org/springframework/cloud/function/adapter/gcloud/integration/CloudFunctionServer.java]: should we use awaitility here?

Comment (dzou) [reply spring-cloud-function-adapters/spring-cloud-function-adapter-gcp/src/test/java/org/springframework/cloud/function/adapter/gcloud/integration/CloudFunctionServer.java]: Good call, done.

**Review by dmitry-s — APPROVED** [review]
LGTM

**Pull request merged**